### PR TITLE
[Work in progress] Fix master

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

**Comment by tsauerwein** [comment]
For reference: This PR aims to fix the build which was broken with https://github.com/camptocamp/ngeo/commit/fe9abceff74ace4e1e0579c99f999f60b8e00df7. That commit was trying to port PR https://github.com/camptocamp/ngeo/pull/1288 and https://github.com/camptocamp/ngeo/pull/1298 from 2.0 to master.

File: contribs/gmf/src/services/querymanager.js [hunk 1]
```diff
@@ -124,7 +124,7 @@ gmf.QueryManager.prototype.createSources_ = function(node) {
         var childLayerNames = [];
         node.childLayers.forEach(function(childLayer) {
           if (childLayer.queryable) {
-            childLayerNames.push(childLayer.layers);
+            childLayerNames.push(childLayer.name);
```
Comment (sbrunner) [inline contribs/gmf/src/services/querymanager.js]: @olivierSemet is this correct?

Comment (tsauerwein) [inline contribs/gmf/src/services/querymanager.js]: At least that is returned by https://geomapfish-demo.camptocamp.net/2.1/wsgi/themes?background=background

**Comment by sbrunner** [comment]
In fact I merge all the 2.0 branch!

**Comment by sbrunner** [comment]
@olivierSemet I merger it, do a new pull request if it's a regression :-)

**Pull request merged**